vmngclient/session.py

```diff
@@ -344,6 +344,20 @@ def get_virtual_session_id(self, tenant_id: str) -> str:
         response = self.post(url_path)
         return response.json()["VSessionId"]
 
+    def logout(self) -> vManageResponse:
+        if self.api_version >= Version("20.12"):
+            return self.post("/logout")
+        else:
+            return self.get("/logout")
+
+    def __enter__(self):
```

Comment:
Do we really need to override `__enter__` and `__exit__`?
I think the better way i just to override `close` method https://requests.readthedocs.io/en/latest/_modules/requests/sessions/#Session.close